### ISSUE TITLE
[Feature]급여명세서를 이용한 교사인증 자동화

### ISSUE THREAD

**Issue by teacherSsamko** [coding-kkanbu/teameet_backend#249]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by hoetaek** [comment]
Is this for the next feature? or do we implement it right away?

**Comment by teacherSsamko** [comment]
No, It's for the version 1.1. we can tell this using Milestones
